src/openloco/interop/hooks.cpp

```diff
@@ -480,9 +487,8 @@ static void register_no_win32_hooks()
     write_jmp(0x4d0fac, (void*)&fn_DirectSoundEnumerateA);
 
     // sound
-    register_hook_stub(0x489cb5);
-    register_hook_stub(0x489f1b);
-    register_hook_stub(0x48a4bf);
```

Comment:
What about this one?

Reply:
It was added in 6f0518f2f82976d8fc13bd2d09ad48a1c8721bcb.

Reply:
Then I don't know what problem I had :/. I don't recall ever using register_hook_stub myself...